cvmfs/download.h

```diff
@@ -138,6 +145,11 @@ struct JobInfo {
     probe_hosts = false;
     head_request = false;
     follow_redirects = false;
+    pid = -1;
```

Comment:
Ah, curl wants to see a file.  Hm, might be worth patching curl to avoid copying credential files around.